.travis.yml

```diff
@@ -21,9 +21,10 @@ addons:
 services:
 - docker
 before_install:
-- docker pull selenium/standalone-chrome:3.8
+- docker pull selenium/standalone-chrome:3.9.1-actinium
 before_script:
-- docker run -d --net=host selenium/standalone-chrome:3.8
+- docker run -d --net=host --shm-size=2g selenium/standalone-chrome:3.8
+- export DBUS_SESSION_BUS_ADDRESS=/dev/null
```

Comment:
just wondering what's that?

Reply:
The `--shm-size=2g` option is as per the recommendation on the selenium/standalone-chrome repo.

As for the DBUS_SESSION_BUS_ADDRESS setting, the selenium docker images actually sets this in it's image due to a bug with selenium. I added it to the setting in case the environment used to run nightmare needed it...I ran the CI tests without the setting and it failed with a timeout every now and then, though after it, it seemed to help. Though this isn't exactly the most scientific approach.